notebooks/benchmark/Default Mask Benchmark.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "trend_plot(benchmarks)"
```

Comment:
for some reason I don't even see the orange line in the first trend plot, any way to fix it/make it obvious which line it is (since there are 3 lines)

Reply:
I guess since they are so similar maybe it doesn't even matter.

Ah, I think I was worried about this notebook instead:
Impute Mask Benchmark.ipynb
here two lines diverge but the third one doesn't appear, so I don't know which one it is more similar to.  However in general they are pretty similar so it doesn't really matter I guess.